### ISSUE TITLE
Botões com CSS

### ISSUE THREAD

**Issue by WillianSugiyama** [frontendbr/forum#1140]
Fala galera, tudo bom? Seguinte, to passando por uma dor de cabeça aqui na empresa, preciso fazer um botão meio baulado(COISA DE UI)

![image](https://user-images.githubusercontent.com/16587394/42292782-5e52e18a-7fab-11e8-899c-30b91d1961a9.png)

![image](https://user-images.githubusercontent.com/16587394/42292789-6a5c4dc2-7fab-11e8-9346-54e250aa08e8.png)

![image](https://user-images.githubusercontent.com/16587394/42292796-746fadc2-7fab-11e8-9d5f-b28ea56e5878.png)

Eu comecei a fazer com before e after, mas não tenho conseguido um resultado como gostaria.

https://pastebin.com/QXgszbE0 -> Código Sass do BTN
https://pastebin.com/3NeHsEr5 -> Código Sass do BTN default

Alguém poderia me ajudar?

**Comment by LFeh** [comment]
@WillianSugiyama O caminho é `before` e `after` mesmo.

Se você observar bem, ele forma um pequeno arco em cima e em baixo, portanto basta criar esse arco e integrar no seu botão.

Tenta colocar seu exemplo no Codepen, fica mais fácil de ver e ajudar :)

**Comment by mateusmiguel** [comment]
bonito.

**Comment by guilhermejcgois** [comment]
Não é algo como [isso](https://codepen.io/guilhermejcgois/pen/bKyeOO) que você precisa? Coloquei a borda em todos os lados dos botões e aumentei um teco as bordas do `before ` e `after`... Não sei se é bem o que você quer, mas me pareceu bem perto

**Comment by WillianSugiyama** [comment]
@guilhermejcgois, a única dificuldade que eu tive desse jeito, seriam os botões vazados, só que já consegui fazer https://codepen.io/willian-sugiyama/pen/qKGNrZ.

Não sabia dessa seguinte propriedade: `border-radius: 400px/40px`

**Comment by guilhermejcgois** [comment]
Nunca tinha usado border-radius com 2 args, bom saber, ficou legal!

**Comment by diogomoretti** [comment]
Sim, dá sim! Referência: https://css-tricks.com/almanac/properties/b/border-radius/
